src/parser/mod.rs

```diff
@@ -30,6 +75,8 @@ pub enum Status {
     Touched,
     /// The key already exists.
     Exists,
+    /// The key already exists and value was requested in the meta protocol.
+    Value,
```

Comment:
This was added because we need a way to differentiate between successful meta get ops where we request the value back with the v flag (e.g. normal get operation) and those where we don't want the value back (e.g. touch to update TTL or a no-op to bump the LRU).